splinterd/api/static/openapi.yml

```diff
@@ -854,3 +1319,28 @@ components:
         prev: /nodes?offset=0&limit=10
         next: /nodes?offset=20&limit=10
         last: /nodes?offset=40&limit=10
+
+    BiomeUserKey:
+      type: object
+      properties:
+        display_name:
+          type: string
+          description: "Display name for key"
+          example: "Biome Admin Key"
+        encrypted_private_key:
+          type: string
+          description: "Encrypted private key"
+          example: "XNzIjoic2VsZi1pc3N1ZWQiLCJleHAiOjE1ODAyMzkyMjh9.P8hA0ru_x\
+            riYX7qryl08ZEp86t5HD_AEVPEUXY70Ehc"
+        public_key:
+          type: string
+          description: "Public key"
+          example: "026c889058c2d22558ead2c61b321634b74e705c42f890e6b7bc2c80abb4713118"
+        user_id:
+          type: string
+          description: "Internal unique identifier for the user"
+          example: "f35aacc1-a9cd-4eda-b6d0-2efaddf0c8a4"
+
+tags:
+  - name: Biome
+    description: Routes supporting user management in splinter applications. Optionally compiled.
```

Comment:
Capitalize splinter if that's consistent with the style used elsewhere (otherwise uncapitalize it elsewhere in this doc). Do we want the api doc to communicate feature status ("optionally compiled")?

Reply:
I wanted to indicate that those routes might not be available, depending on how the daemon was set up. I can remove that entirely, I'm also open to better wording suggestions.